Jenkinsfile

```diff
@@ -707,7 +707,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
           error("Error '${err.toString()}'")
         } finally {
           if (archive) {
-            archiveArtifacts(allowEmptyArchive: true, artifacts: "${directory}/build/system-tests/docker-logs/TEST-docker-compose-*.log")
+            archiveArtifacts(allowEmptyArchive: true, artifacts: "${directory}/build/system-tests/docker-logs/TEST-docker-compose-*.log, ${directory}/build/integration-tests/**/**")
```

Comment:
Was it just for debugging or we still need it?

Reply:
If this test ever fails on CI that will be needed. I believe we should keep it there. Nobody investigating a test failure will remember to re-enable that, run it again then disable it.

I'm trying to make it as easy as possible to find and fix anything that could make this test fail.

Reply:
Do the tests output the junit test report that CI uses to generate the test report? The unit tests do this, there probably isn't a reason the integration tests here couldn't either.

This is all controlled through the mage Go Test wrapper so if you are using that you are set.

https://github.com/belimawr/beats/blob/d36f48672653082fd5a8e27795726fb87c733be2/dev-tools/mage/gotest.go#L211

Reply:
I don't get how it could help. What I am trying to do here is to make the logs from  the Filebeat test binary available to download on CI, so if the test fails we can see why Filebeat failed/did not behave as expected.